src/components/SharedStyledComponents.js

```diff
@@ -443,3 +443,27 @@ export const dropdownIconContainerVariant = {
   },
   closed: { rotate: -90, y: 0 },
 }
+
+// Common styled item card for languages/about
+
+export const CardItem = styled(Link)`
+  text-decoration: none;
+  margin: 1rem 1rem 1rem 0;
+  padding: 1rem;
+  flex: 1 1 240px;
```

Comment:
This causes the bottom row of the Translations page to have ~1-3 cards that get stretched to fill the whole row (depending on screen width).
`flex: 0 1 240px;` should prevent this